libs/application/templates/financial-aid/src/lib/dataSchema.ts

```diff
@@ -72,6 +72,11 @@ export const dataSchema = z.object({
       params: error.validation.inputErrorMessage,
       path: ['custom'],
     }),
+  bankInfoForm: z.object({
+    bankNumber: z.string().optional(),
+    ledger: z.string().optional(),
+    accountNumber: z.string().optional(),
+  }),
```

Comment:
Have you tried the validation with every field empty? Does that work?

Reply:
yess it works.. i think, at least in locally